PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript.rbxmx

```diff
@@ -138,6 +138,8 @@ end
 local function Update()
 	if EnabledCamera then
 		EnabledCamera:Update()
+		EnabledCamera:ApplyCameraCFrame()
+		EnabledCamera:ApplyVROffset()
```

Comment:
It's a Lua function, you can find it in RootCamera. It basically just applies the UserHeadCFrame on top of the current camera CFrame, but the plan was for individual cameras to modify it for more special handling.

I would love to see these camera modules become more object-oriented. Right now most of them just repeat the same code.